WalletWasabi.Fluent/WalletWasabi.Fluent.csproj

```
@@ -19,10 +19,10 @@
 		<AvaloniaResource Include="Assets\**" />
 	</ItemGroup>
 	<ItemGroup>
-		<PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
-		<PackageReference Include="Avalonia.ReactiveUI" Version="$(AvaloniaVersion)" />
-		<PackageReference Include="Avalonia.Controls.DataGrid" Version="$(AvaloniaVersion)" />
-		<PackageReference Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)" Condition="'$(Configuration)' == 'Debug'" />
+		<PackageReference Include="Avalonia" Version="0.10.11-rc.1" />
+		<PackageReference Include="Avalonia.ReactiveUI" Version="0.10.11-rc.1" />
+		<PackageReference Include="Avalonia.Controls.DataGrid" Version="0.10.11-rc.1" />
+		<PackageReference Include="Avalonia.Diagnostics" Version="0.10.11-rc.1"/>
```

Is there a reason to remove `$(AvaloniaVersion)`?

I would expect:

```suggestion
		<PackageReference Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)"/>
```

---

No, it was a mistake.